content/documentation/pages/6-recipes/2-file-ingest/1-sftp-to-jdbc.md

```diff
@@ -0,0 +1,936 @@
+We also require an external SFTP server with a `/remote-files` directory.
+
+#### Register the applications
+
+If you downloaded the [sample project](https://github.com/spring-cloud/spring-cloud-dataflow-samples/blob/master/dataflow-website/recipes/file-ingest/file-to-jdbc/file-to-jdbc.zip?raw=true) you can build and publish the docker image to the minikube registry:
+
+```bash
+$eval $(minikube docker-env)
+$./mvnw clean package docker:build -Pkubernetes
+```
+
+Otherwise, you can skip this step to pull the image from dockerhub.
+
+```bash
+app register --name fileIngest --type task --uri docker://springcloud/ingest
+```
+
+Register the prepackaged `sftp` source and `task-launcher` sink applications:
+
+```bash
+app register --name sftp --type source  --uri docker://springcloudstream/sftp-dataflow-source-kafka:2.1.0.RELEASE --metadata-uri maven://org.springframework.cloud.stream.app:sftp-dataflow-source-kafka:jar:metadata:2.1.0.RELEASE
+```
+
+```bash
+app register --name task-launcher --type sink --uri docker://springcloudstream/task-launcher-dataflow-sink-kafka:1.0.1.RELEASE --metadata-uri maven://org.springframework.cloud.stream.app:task-launcher-dataflow-sink-kafka:jar:metadata:1.0.1.RELEASE
+```
+
+#### Create the task
+
+```bash
+task create fileIngestTask --definition fileIngest
+```
+
+#### Create the stream
+
+The `sftp` source is configured to publish a task launch request to launch the `fileIngestTask` task.
+The launch request mounts the nfs share to the task pod using deployment properties
+`deployer.*.kubernetes.volumes=[{'name':'staging','persistentVolumeClaim':{'claimName':'nfs-volume-claim'}}]` and
+`deployer.*.kubernetes.volumeMounts=[{'mountPath':'/staging/shared-files','name':'staging'}]`.
+
+<!--NOTE-->
+
+**NOTE**: Replace `<user>`, `<pass>` and `<data-flow-server-uri>` in the stream definition below. The `<host>` value here is the default minikube gateway for VirtualBox.
+
+To get the `<data-flow-server-uri>` find the name of the service and use the `minikube service` command:
+
+```bash:
+$kubectl get svc
+...
+my-release-data-flow-server     NodePort       10.97.74.123     <none>        80:30826/TCP
+
+$minikube service my-release-data-flow-server --url
+http://192.168.99.105:30826
+```
+
+<!--END_NOTE-->
+
+```bash
+stream create inboundSftp --definition "sftp --host=192.168.99.1 --username=<user> --password=<pass> --allow-unknown-keys=true --remote-dir=/remote-files --local-dir=/staging/shared-files --task.launch.request.taskName=fileIngestTask --task.launch.request.deployment-properties=deployer.*.kubernetes.volumes=[{'name':'staging','persistentVolumeClaim':{'claimName':'nfs-volume-claim'}}],deployer.*.kubernetes.volumeMounts=[{'mountPath':'/staging/shared-files','name':'staging'}] | task-launcher --spring.cloud.dataflow.client.server-uri=<dataflow-uri>"
```

Comment:
Should we describe `<dataflow-uri>` in a NOTE?

Reply:
This is one of the points that should be in the install docs. Here we assume they know what this is.